### PR TITLE
Update/test secrect-contracts.md and privacy-model-of-secret-contracts.md

### PR DESCRIPTION
**Pull request by StoicNorth**
Tested code snippets up to 'Simple Counter Project'. Also made minor copy updates. @dylanschultzie

### CODE REVIEW

**Comment by StoicNorth** [comment]
Will make separate commits directly to the phase 2 branch after this PR is accepted @dylanschultzie

**Comment by StoicNorth** [comment]
Complete working through secret-contracts.md

**Comment by StoicNorth** [comment]
Updated copy and test code of secret-contracts.md and privacy-model-of-secret-contracts.md @dylanschultzie

File: docs/dev/privacy-model-of-secret-contracts.md [hunk 1]
```diff
@@ -361,28 +361,28 @@ Types of external `query`:
...
-Most of the below examples talk about an attacker revealing which function was executed on the contract, but this is not the only type of data leakage that an attacker might target.
+Most of the below examples talk about an attacker revealing which function was executed on the contract, but this is not the only type of data leakage attackers may target.
 
 Secret Contract developers must analyze the privacy model of their contract - What kind of information must remain private and what kind of information, if revealed, won't affect the operation of the contract and its users. **Analyze what it is that you need to keep private and structure your Secret Contract's boundaries to protect that.**
 
 ## Differences in input sizes
 
 An example input API for a contract with 2 `handle` functions:
 
-```rust
```
Comment (dylanschultzie) [inline docs/dev/privacy-model-of-secret-contracts.md:385]: we do want these in here - it's what gives it the formatting.

File: docs/dev/privacy-model-of-secret-contracts.md [hunk 2]
```diff
@@ -404,7 +404,7 @@ These inputs are encrypted, but by looking at their size an attacker can guess w
 
 A quick fix for this issue might be renaming `Transfer` to `Tsfr`:
 
-```rust
```
Comment (dylanschultzie) [inline docs/dev/privacy-model-of-secret-contracts.md:407]: and here

File: docs/dev/privacy-model-of-secret-contracts.md [hunk 3]
```diff
@@ -429,20 +429,20 @@ Another point to consider. If the attacker had additional knowledge, for example
...
+Again, this is not recommended because you cannot guarantee control of the client!
 
 ## Differences in state key sizes
 
-Contracts' state is stored on-chain inside a key-value store, thus the `key` must remain constant between calls. This means that if a contract uses storage keys with different sizes, an attacker might find out information about the execution of a contract.
+Contracts' state is stored on-chain inside a key-value store where the `key` must remain constant between calls. This means if a contract uses storage keys with different sizes, an attacker might find out information about the execution of a contract.
 
 Let's see an example for a contract with 2 `handle` functions:
 
-```rust
```
Comment (dylanschultzie) [inline docs/dev/privacy-model-of-secret-contracts.md:445]: and here

File: docs/dev/privacy-model-of-secret-contracts.md [hunk 4]
```diff
@@ -468,17 +468,17 @@ pub fn handle<S: Storage, A: Api, Q: Querier>(
...
 2. `transfer`
 
 Again, some quick fixes for this issue might be:
 
-1. Renaming `transfer` to `tsfr`.
-2. Padding `send` to have the same length as `transfer`: `sendsend`.
+1. Renaming `transfer` to `tsfr`
+2. Padding `send` to have the same length as `transfer`: `sendsend`
 
-```rust
```
Comment (dylanschultzie) [inline docs/dev/privacy-model-of-secret-contracts.md:481]: and here

File: docs/dev/privacy-model-of-secret-contracts.md [hunk 5]
```diff
@@ -512,7 +512,7 @@ Very similar to the state key sizes case, if a contract uses storage values with
 
 Let's see an example for a contract with 2 `handle` functions:
 
-```rust
```
Comment (dylanschultzie) [inline docs/dev/privacy-model-of-secret-contracts.md:515]: and here

File: docs/dev/privacy-model-of-secret-contracts.md [hunk 6]
```diff
@@ -544,17 +544,17 @@ pub fn handle<S: Storage, A: Api, Q: Querier>(
...
 2. `Transferred amount: 123`
 
 Again, some quick fixes for this issue might be:
 
-1. Changing the `Transferred` string to `Tsfr`.
-2. Padding `Sent` to have the same length as `Transferred`: `SentSentSen`.
+1. Changing the `Transferred` string to `Tsfr`
+2. Padding `Sent` to have the same length as `Transferred`: `SentSentSen`
 
-```rust
```
Comment (dylanschultzie) [inline docs/dev/privacy-model-of-secret-contracts.md:557]: (and more further down, but I'll stop here)

**Review by dylanschultzie — CHANGES_REQUESTED** [review]

**Review by StoicNorth — COMMENTED** [review]
Fixed formatting

File: docs/dev/privacy-model-of-secret-contracts.md [hunk 7]
```diff
@@ -468,17 +468,17 @@ pub fn handle<S: Storage, A: Api, Q: Querier>(
...
 
 Again, some quick fixes for this issue might be:
 
-1. Renaming `transfer` to `tsfr`.
-2. Padding `send` to have the same length as `transfer`: `sendsend`.
+1. Renaming `transfer` to `tsfr`
+2. Padding `send` to have the same length as `transfer`: `sendsend`
 
-```rust
+```rustrust
```
Comment (dylanschultzie) [inline docs/dev/privacy-model-of-secret-contracts.md:481]: typo.

there's got to be some way that I can simply go in and fix these small errors.

**Pull request merged**